src/index.html

```diff
@@ -17,34 +19,82 @@ <h2 class="gallery-title">Our Meetups Gallery</h2>
             <p class="card-title">August RoundTable Meetup</p>
-            <p class="card-text card-secondary-text text-supporting"><img class="map-icon" src="images/map-pin.svg"> Bo's Coffee, Ayala</p>
+            <p class="card-text card-secondary-text text-supporting"><img class="map-icon" src="images/map-pin.svg">
+              Bo's Coffee, Ayala</p>
           </div>
         </div>
 
         <div class="card gallery-grid-rightCol">
           <img class="card-photo" src="images/meetup-july.jpg" alt="meetup-july">
           <div class="card-text-container">
             <p class="card-title">July RoundTable Meetup</p>
-            <p class="card-text card-secondary-text text-supporting"><img class="map-icon" src="images/map-pin.svg"> Coffee Project, Mandaluyong</p>
+            <p class="card-text card-secondary-text text-supporting"><img class="map-icon" src="images/map-pin.svg">
+              Coffee Project, Mandaluyong</p>
           </div>
         </div>
-        
+
         <div class="card gallery-grid-leftCol">
           <img class="card-photo" src="images/meetup-june.jpg" alt="meetup-june">
           <div class="card-text-container">
             <p class="card-title">June RoundTable Meetup</p>
-            <p class="card-text card-secondary-text text-supporting"><img class="map-icon" src="images/map-pin.svg"> Coffee Project, Mandaluyong</p>
+            <p class="card-text card-secondary-text text-supporting"><img class="map-icon" src="images/map-pin.svg">
+              Coffee Project, Mandaluyong</p>
           </div>
         </div>
 
         <div class="card gallery-grid-rightCol">
           <img class="card-photo" src="images/meetup-february.jpg" alt="meetup-february">
           <div class="card-text-container">
             <p class="card-title">February Coffee + Code Meetup</p>
-            <p class="card-text card-secondary-text text-supporting"><img class="map-icon" src="images/map-pin.svg"> Bo's Coffee Primo, Mandaluyong</p>
+            <p class="card-text card-secondary-text text-supporting"><img class="map-icon" src="images/map-pin.svg">
+              Bo's Coffee Primo, Mandaluyong</p>
           </div>
         </div>
       </div>
     </section>
+    <footer>
+      <div class="row footer-container">
+        <div class="column">
+          <h3 class="footer-header"><img src="images/freeCodeCamp-logo.png">freeCodeCamp Manila</h3>
+          <p class="footer-support-text">We are not affiliated with freeCodeCamp.org and are an independent local community and study group.</p>
+          <p class="footer-support-text-owned">freeCodeCamp Logo is owned by freeCodeCamp.org</p>
+          <small>
+            &copy 2019 freeCodeCamp Manila
+          </small>
+        </div>
+        <div class="column column-2">
+          <h2 class="small-footer-header">freeCodeCamp</h2>
+          <p class="footer-support-text">
+            <a href="https://learn.freecodecamp.org/">Curriculum</a>
```

Comment:
For all the links - `a` tags - please add `target="_blank" rel="noopener noreferrer"`